addons/l10n_it/data/account.tax.template.csv

```diff
@@ -82,3 +39,15 @@ id,description,chart_template_id/id,name,sequence,amount,amount_type,type_tax_us
 22I5b,22I5b,l10n_it_chart_template_generic,IVA al 22% detraibile al 50% (D),200,11,percent,none,False,1601,1601,,,tax_group_iva_22_de_50
 22I5a,22I5a,l10n_it_chart_template_generic,IVA al 22% detraibile al 50% (I),100,11,percent,none,False,,,,,tax_group_iva_22_de_50
 22I5,22I5,l10n_it_chart_template_generic,IVA al 22% detraibile al 50%,35,22,group,purchase,False,,,tax_tag_19,"22I5b,22I5a",tax_group_iva_22_de_50
+00art8cv,00art8cv,l10n_it_chart_template_generic,Non imponibile Art.8c Lettera Intento,40,0,percent,sale,False,2601,2601,tax_tag_46,,tax_group_imp_ese_art_8
+00art2c3lAa,00art2c3lAa,l10n_it_chart_template_generic,Fuori campo IVA Art.2 Com.3 Lett.A DPR 633/72,40,0,percent,purchase,False,1601,1601,tax_tag_47,,tax_group_imp_ese_art_2
+00art9a,00art9a,l10n_it_chart_template_generic,Non imponibile Art.9,40,0,percent,purchase,False,1601,1601,tax_tag_48,,tax_group_imp_ese_art_9
+00art41v,00art41v,l10n_it_chart_template_generic,Non imponibile Art.41 D.L.331/93,40,0,percent,sale,False,2601,2601,tax_tag_49,,tax_group_imp_ese_art_41
+00art71v,00art71v,l10n_it_chart_template_generic,Non imponibile Art.71 DPR.633/72,40,0,percent,sale,False,2601,2601,tax_tag_50,,tax_group_imp_ese_art_71
+00art74v,00art74v,l10n_it_chart_template_generic,Imponibile Escluso Art.74 DPR 633/72,40,0,percent,sale,False,2601,2601,tax_tag_51,,tax_group_imp_ese_art_74
+00art7c4a,00art7c4a,l10n_it_chart_template_generic,Fuori campo IVA Art.7 comm.4,40,0,percent,purchase,False,1601,1601,tax_tag_52,,tax_group_imp_ese_art_7
+00art10a,00art10a,l10n_it_chart_template_generic,Imponibile Esente Art.10,40,0,percent,purchase,False,1601,1601,tax_tag_53,,tax_group_imp_ese_art_10
+00art58v,00art58v,l10n_it_chart_template_generic,Non imponibile Art.58,40,0,percent,sale,False,2601,2601,tax_tag_54,,tax_group_imp_ese_art_58
+00art8v,00art8v,l10n_it_chart_template_generic,Non imponibile Art.8,40,0,percent,sale,False,2601,2601,tax_tag_55,,tax_group_imp_ese_art_8
+00art196a,00art196a,l10n_it_chart_template_generic,Art.196 Dir.2006/112/EC (credito),40,0,percent,purchase,False,1601,1601,tax_tag_56,,tax_group_imp_ese_art_196
+00art196v,00art196v,l10n_it_chart_template_generic,Art.196 Dir.2006/112/EC (debito),40,0,percent,sale,False,2601,2601,tax_tag_57,,tax_group_imp_ese_art_196
```

Comment:
Imponibile Escluso → Imponibile escluso
Imponibile Esente → Imponibile esente
Dir. → dir.
Art. → art.

Others lines, not changed by this PR, need to be checked too.